src/net_processing.cpp

```diff
@@ -3811,15 +3837,14 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             vRecv >> nonce;
 
             // Only process pong message if there is an outstanding ping (old ping without nonce should never pong)
-            if (pfrom.nPingNonceSent != 0) {
-                if (nonce == pfrom.nPingNonceSent) {
+            if (peer->nPingNonceSent != 0) {
+                if (nonce == peer->nPingNonceSent) {
                     // Matching pong received, this ping is no longer outstanding
                     bPingFinished = true;
-                    const auto ping_time = ping_end - pfrom.m_ping_start.load();
-                    if (ping_time.count() >= 0) {
-                        // Successful ping time measurement, replace previous
-                        pfrom.nPingUsecTime = count_microseconds(ping_time);
-                        pfrom.nMinPingUsecTime = std::min(pfrom.nMinPingUsecTime.load(), count_microseconds(ping_time));
+                    const auto ping_time = ping_end - peer->m_ping_start.load();
+                    if (ping_time.count() > 0) {
```

Comment:
just noting that this changed from `>=` to `>`. makes sense to call it a "timing mishap" if the pong was received in the same microsecond as we recorded for the ping going out, so seems fine. just highlighting the change since it's easy to miss.

Reply:
Oops, this was unintentional. You're right that it shouldn't have any impact, but I'm going to revert it to how it was. Good catch!